packages/react-components/src/components/TypingIndicator.tsx

```diff
@@ -70,12 +72,12 @@ const getDefaultComponents = (
         className={mergeStyles(typingIndicatorListStyle, styles?.typingUserDisplayName)}
         key={`typing indicator display string ${index}`}
       >
-        {index < typingUsers.length - 1 ? typingUser.displayName + ', ' : typingUser.displayName}
+        {index < typingUsersToRender.length - 1 ? typingUser.displayName + ', ' : typingUser.displayName}
       </span>
     );
   });
 
-  const countOfUsersNotMentioned = typingUsers.length - typingUsersMentioned.length;
+  const countOfUsersNotMentioned = typingUsersToRender.length - typingUsersMentioned.length;
```

Comment:
Should this continue to use `typingUsers`? It makes sense to not name participants without a `displayName` property but they should be included in the count of number of participants that are typing.